src/advanced_inv_area.cpp

```diff
@@ -216,115 +205,94 @@ bool advanced_inv_area::is_same( const advanced_inv_area &other ) const
 
 bool advanced_inv_area::canputitems( const advanced_inv_listitem *advitem )
```

Comment:
I'm unsure of what to do with this method. 
Realistically I'd like to change the defintition to remove the parameter then just inline it to return `canputitemsloc` but I didn't want to make changes I didn't need to make.

I'd like some feedback if possible, thanks.